underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -270,50 +369,61 @@ public InputStream open(String path) throws IOException {
   }
 
   /**
-   * Each path is checked both for leading "/" and ending "/". Leading "/" is removed, and "/" is
-   * added at the end if not present.
+   * A trailing {@link SwiftUnderFileSystem#PATH_SEPARATOR} is added if not present.
    *
    * @param path URI to the object
-   * @return qualified path
+   * @return folder path
    */
-  private String makeQualifiedPath(String path) {
-    if (!path.endsWith("/")) {
-      path = path + "/";
-    }
-    if (path.startsWith("/")) {
-      path = path.substring(1);
-    }
-    return path;
+  private String addFolderSuffixIfNotPresent(String path) {
```

Comment:
Why do some methods call `addFolderSuffixIfNotPresent` and some call `PathUtils.normalizePath` ?